code/modules/paperwork/paper.dm

```diff
@@ -618,3 +618,91 @@
 
 /obj/item/weapon/paper/crumpled/bloody
 	icon_state = "scrap_bloodied"
+
+/obj/item/weapon/paper/evilfax
+	name = "Centcomm Reply"
+	info = ""
+	var/mytarget = null
+	var/myeffect = null
+	var/used = 0
+	var/countdown = 60
+	var/activate_on_timeout = 0
+
+/obj/item/weapon/paper/evilfax/show_content(var/mob/user, var/forceshow = 0, var/forcestars = 0, var/infolinks = 0, var/view = 1)
+	if (user == mytarget)
+		if (istype(user, /mob/living/carbon))
+			var/mob/living/carbon/C = user
+			evilpaper_specialaction(C)
+			..()
+		else
+			// This should never happen, but just in case someone is adminbussing
+			evilpaper_selfdestruct()
+	else
+		if (mytarget)
+			to_chat(user,"<span class='notice'>This page appears to be covered in some sort of bizzare code. The only bit you recognize is the name of [mytarget]. Perhaps [mytarget] can make sense of it?</span>")
+		else
+			evilpaper_selfdestruct()
+
+
+/obj/item/weapon/paper/evilfax/New()
+	..()
+	processing_objects += src
+
+
+/obj/item/weapon/paper/evilfax/Destroy()
+	processing_objects -= src
+	if (mytarget && !used)
+		var/mob/living/carbon/target = mytarget
+		target.ForceContractDisease(new /datum/disease/transformation/corgi(0))
+	return ..()
+
+
+/obj/item/weapon/paper/evilfax/process()
+	if (countdown == 0)
+		if (mytarget)
+			if (activate_on_timeout)
+				evilpaper_specialaction(mytarget)
+			else
+				message_admins("[mytarget] ignored an evil fax until it timed out.")
+		else
+			message_admins("Evil paper '[src]' timed out, after not being assigned a target.")
+		used = 1
+		evilpaper_selfdestruct()
+	else
+		countdown--
+
+	return
+
+/obj/item/weapon/paper/evilfax/proc/evilpaper_specialaction(var/mob/living/carbon/target)
+	if (myeffect == "borging")
+		target.ForceContractDisease(new /datum/disease/transformation/robot(0))
+	else if (myeffect == "corgifying")
+		target.ForceContractDisease(new /datum/disease/transformation/corgi(0))
+	else if (myeffect == "explosive")
+		var/turf/simulated/floor/T = get_turf(target)
+		if (istype(T))
+			T.break_tile_to_plating()
+		var/obj/effect/stop/S
+		S = new /obj/effect/stop
+		S.victim = target
+		S.loc = target.loc
+		spawn(20)
+			qdel(S)
+		target.adjustBruteLoss(target.health)
+	else if (myeffect == "memetickillagent")
+		to_chat(target,"<span class='userdanger'>A series of bright lights flash across your vision: MEMETIC KILL AGENT YHWH-3 ACTIVATED</span>")
```

Comment:
This is what His Grace gives. Let's not generate confusion by having two things with different names.